trino/sqlalchemy/compiler.py

```diff
@@ -181,7 +181,14 @@ def visit_try_cast(self, element, **kw):
 
 
 class TrinoDDLCompiler(compiler.DDLCompiler):
-    pass
+    def visit_foreign_key_constraint(self, constraint, **kw):
+        return None
```

Comment:
Rather than silently ignoring constraints, maybe we should raise an explicit error indicating that these constraints are not supported in Trino:
```suggestion
from sqlalchemy.exc import CompileError
...
    def visit_foreign_key_constraint(self, constraint, **kw):
        raise CompileError("Trino does not support FOREIGN KEY constraints.")
```
___
Or at least we could add a warning to make it clearer that the constraint is being ignored.
```suggestion
import warnings
from sqlalchemy.exc import SAWarning
...
    def visit_foreign_key_constraint(self, constraint, **kw):
        warnings.warn("Trino does not support FOREIGN KEY constraints. Constraint will be ignored.", SAWarning)
        return None
```

@willmostly @hashhar WDYT?

Reply:
@willmostly PTAL.